common/error.go

```diff
@@ -19,9 +19,8 @@ const (
 	NotImplemented Code = 6
 
 	// 101 ~ 199 db error
-	DbConnectionFailure    Code = 101
-	DbStatementSyntaxError Code = 102
-	DbExecutionError       Code = 103
+	DbConnectionFailure Code = 101
```

Comment:
Since we are using namespacing, the error code can overlap between different namespaces (also it's hard to avoid overlapping since different namespaces are not aware of each other) We can do this in a separate PR.

Reply:
Sure, I updated the error code to make them sequence